apps/omg/test/omg/state/core_test.exs

```diff
@@ -56,9 +65,9 @@ defmodule OMG.State.CoreTest do
       # make some utxos
       state =
         state
-        |> Core.exec(create_recovered([{1, 0, 0, alice}], @eth, [{alice, 6}, {bob, 2}, {alice, 2}]), :ignore_fees)
```

Comment:
hm, but `:ignore_fees` as an option to `Core.exec` still must be supported, since there are clients of `State.Core` that don't check the fees being paid (the Watcher). I see you're leaving `core.ex` without modification, so I suppose the tests of it should be OK without change.

In other words - we're only removing `ignore_fees: true/false` setting, which removes the possibility to ignore _for the child chain server_ only, correct?

Reply:
Correct, `Core.exec` still supports `ignore_fees` only the config has been removed. Let me check and add a test if needed with `ignore_fees` to the Core.